custom_components/frigate/camera.py

```diff
@@ -153,24 +155,28 @@ def __init__(
             self._cam_name
             in self._frigate_config.get("go2rtc", {}).get("streams", {}).keys()
         ):
-            self._restream_type = "rtsp"
-            streaming_template = config_entry.options.get(
-                CONF_RTSP_URL_TEMPLATE, ""
-            ).strip()
-
-            if streaming_template:
-                # Can't use homeassistant.helpers.template as it requires hass which
-                # is not available in the constructor, so use direct jinja2
-                # template instead. This means templates cannot access HomeAssistant
-                # state, but rather only the camera config.
-                self._stream_source = Template(streaming_template).render(
-                    **self._camera_config
-                )
+            if config_entry.options.get(CONF_ENABLE_WEBRTC, False):
+                self._restream_type = "webrtc"
```

Comment:
this is incorrect, restream type refers to what type of stream from the frigate side is being supplied